README.md

```diff
@@ -22,56 +22,47 @@ vars:
 ```
 
 ## Contents
-This dbt package contains macros for SQL functions to run the dbt project on multiple databases and for generic tests. The databases that are currently supported are Snowflake, SQL Server and Databricks.
+This dbt package contains macros for SQL functions to run the dbt project on multiple databases. The databases that are currently supported are Snowflake, SQL Server and Databricks.
 
-- [Multiple databases](#Multiple-databases)
-  - [as_varchar](#as_varchar-source)
-  - [charindex](#charindex-source)
+- [SQL generators](#SQL-generators)
```

Comment:
I ordered them according to how important/frequently used they seem to me. Otherwise, the "aggregate functions" appear on top, and these are probably the least used. 

I had the same concern about ordering not matching between readme and groups. However, the main folder is called "macros", so you even need to click one folder further to see the groups as they appear in the readme.

So, my preference is ordering on importance instead of alphabetically since that seem to have more value for the user.

Reply:
Ok, let's go with this order for now. It does mean we need to reconsider the importance every time we add another group, but maybe that's alright since hopefully we do not do that, that often.